### ISSUE TITLE
Theme language files not working

### ISSUE THREAD

**Issue by maxlysenko** [getgrav/grav#357]
Themes `languages.yaml` files are not loaded.
From [`Themes.php`](https://github.com/getgrav/grav/blob/develop/system/src/Grav/Common/Themes.php#L233) I guess some prefixes in theme configuration are needed
```php
$schemes = array_reverse($config->get("themes.{$name}.streams.schemes.theme.prefixes.", []));
```
but in the theme configuration there are only paths 
```yaml
streams:
  schemes:
    theme:
      type: ReadOnlyStream
      paths:
        - ...
```
Maybe it shoud be `$config->get("themes.{$name}.streams.schemes.theme.paths", [])` instead, or where to put these prefixes and what they should contain?

**Comment by rhukster** [comment]
@maxlysenko are you referring to the updates made in Develop yesterday?

**Comment by Sommerregen** [comment]
@rhukster He is referring to the latest changes.

@maxlysenko I implemented theme inheritance according to the blog post http://getgrav.org/blog/theme-development-with-inheritance, which uses prefixes. Paths are currently not supported. 

But you are right. I forgot to setup a default fallback, if no streams are given in the yaml file... sorry :disappointed: No need to add something to the theme configurations. Instead please change line [233](https://github.com/getgrav/grav/blob/develop/system/src/Grav/Common/Themes.php#L233) to

```
$schemes = array_reverse($config->get("themes.{$name}.streams.schemes.theme.prefixes.", ["themes://{$name}"]));
```

and come back if it not works.

**Comment by rhukster** [comment]
That loads the theme's languages.yaml file if it has one, but does not find any parent language files. 

So if there is no language file in the child theme, nothing is loaded or merged.

**Comment by rhukster** [comment]
Actually i think max is onto something.. hold on testing...

**Comment by Sommerregen** [comment]
@rhukster Yes, you are right. It is not optimal. But child themes should always set up a streams path like in the blog post. Otherwise they are not treated as child themes of a parent one.

**Comment by rhukster** [comment]
Confusingly, my test child theme was also using `paths:` not `prefixes:`.  Now I have to work out why/how  I was using this (as was Max) as it's not documented that way in the blog post or in the docs!

**Comment by Sommerregen** [comment]
@maxlysenko The issue should be fixed with commit [2a6437a](https://github.com/Sommerregen/grav/commit/2a6437a2f1b67c37976cfeaedfd00b2d2b62061c) and thank you for pointing out the bug/problem with loading languages using theme inheritance!
